### PR TITLE
Don't match utm_ when it's in the middle of a key

### PR DESCRIPTION
**Pull request by dy-dx**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -3,12 +3,8 @@ describe('utm-params', function(){
   var assert = require('component/assert');
   var utm = require('../index');
 
-  location.__defineGetter__('search', function(){
```
Comment (ScotterC) [inline test/test.js]: @dy-dx why rid yourself of this abstraction?

Comment (dy-dx) [inline test/test.js]: It doesn't work in Chrome for me.

Comment (ScotterC) [inline test/test.js]: huh weird.  some esoteric JS issue then :)

**Comment by sperand-io** [comment]
thanks! this was closed by #3 :) https://github.com/segmentio/utm-params/pull/3/files#diff-0fd0e07cf6d02bf7cf00f18cebb8e6eaR27

**Comment by dy-dx** [comment]
@sperand-io great, thank you! Now to follow the dependency tree upwards... https://github.com/segment-integrations/analytics.js-integration-segmentio/issues/2

**Comment by sperand-io** [comment]
@dy-dx yessir, though I'm *hoping* to get [this](https://github.com/segmentio/utm-params/pull/5) merged, released and adopted upstream in a.js, so as to bound the possible utm params we'll send downstream (and to redshift, s3, etc) to the standard ones!

**Pull request closed without merging**